src/providers/mod.rs

```diff
@@ -64,6 +67,23 @@ fn create_file(filename: &str) -> Result<File> {
     File::create(file_path).chain_err(|| format!("failed to create file {:?}", file_path))
 }
 
+fn write_structured_journald_entry(
```

Comment:
```suggestion
/// Add a message to the journal logging SSH key additions; this
/// will be used by at least Fedora CoreOS to display in the console
/// if no ssh keys are present.
fn write_ssh_key_journal_entry(
```

Reply:
I think this is worth a comment.

Also, right now it's a bit odd because it takes the message as a parameter, but hardcodes the message ID.  Either we're making a generic function, or we're doing something just for SSH.  I think the latter makes more sense, so that's the rationale for renaming the method.